### DISCUSSION TITLE
Hi guys from the OT of osu forum

### DISCUSSION THREAD

**Discussion by tematikkp** [TheOriginalAyaka/sekai-stickers#12] [General]
This is THETM speaking, and as of February 8, 2023, I'm using this pfp w. This is because there is a disease in the Off-Topic forums, and I'm going to treat my current profile picture there as a "collab". I'm not either a weeb or an otaku btw.

**Comment by TheOriginalAyaka** [comment]
> This is THETM speaking, and as of February 8, 2023, I'm using this pfp w. This is because there is a disease in the Off-Topic forums, and I'm going to treat my current profile picture there as a "collab". I'm not either a weeb or an otaku btw.

( ・◇・)？what

**Reply by tematikkp** [reply]
https://osu.ppy.sh/community/forums/topics/1718633 here (I'm not the original poster)

**Comment by Corne2Plum3** [comment]
lmao

**Comment by Jarcrafted-Stuff** [comment]
Ain't no way bruh

**Comment by fakeaireuonboard** [comment]
made an account just to post here 😯

**Comment by keremaru** [comment]
### **^Same, but also to gage reception on a counter-collab to combat the ProSekai collab. Thinking about Azur Lane or KanColle.**

**Comment by 179bpm** [comment]
a bit late but 10/10 very wonderhoy would be possesed by honami again (yeah i did that)

also the thread got locked/archived :(
